components/header/Alert.tsx

```diff
@@ -1,32 +1,58 @@
 import Slider from "$store/components/ui/Slider.tsx";
 import SliderJS from "$store/islands/SliderJS.tsx";
 import { useId } from "$store/sdk/useId.ts";
+import { Color } from "https://deno.land/x/color@v0.3.0/mod.ts";
 
-export interface Props {
-  alerts: string[];
+export interface barColor {
+  /**
+   * @format color
+   * @title Background
+   * @default #121212
+   */
+  background: string;
```

Comment:
Nao é necessario colocar title se é o mesmo que o nome da variavel
```suggestion
   * @default #121212
   */
  background: string;
```